handlers/load_test_handler.go

```diff
@@ -419,7 +427,7 @@ func (h *Handler) executeLoadTest(ctx context.Context, req *http.Request, testNa
 
 	tokenVal, _ := provider.GetProviderToken(req)
 
-	logrus.Debugf("promURL: %s, testUUID: %s, resultID: %s", promURL, testUUID, resultID)
+	h.log.Debugf("promURL: %s, testUUID: %s, resultID: %s", promURL, testUUID, resultID)
```

Comment:
`h.log.Debug` uses `logrus.Log` internally, and not `logrus.Logf`.

you might want to rewrite it as
Consider writing it as `h.log.Debug("promURL: , testUUID: , resultID: ", promURL, testUUID, resultID)`

Reply:
do i convert all the Debugf to Debug?

Reply:
Yes.